code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -222,9 +222,15 @@
 	else //Upgrade process finished or impossible
 		stat("Upgrade Progress:", "(FINISHED)")
 
+	if(xeno_caste.max_health > 0)
```

Comment:
all castes have max_health, so this is useless

Reply:
all xenos have sunder, even larva and hivemind. I dunno how to exclude them, so I'm just gonna leave it as is